src/timeline/js/controllers.js

```diff
@@ -1408,19 +1402,6 @@ $scope.SetTrackLabel = function (label){
 	 // return true
 	 return true;
  };
-
- // Lock and unlock items
- $scope.LockItems = function(){
-
-	// Enable all items
-	//$(".clip").draggable("enable")
-
-	// Disable any locked items
-	// for (layer in $scope.project.layers)
-	// {
-	// 	timeline.qt_log(layer);
-	// }
- };
```

Comment:
What _will_ we do without this function??

(Which reminds me, one of us should just delete `src/timeline/media/js/debug.js` entirely, as it contains equally critical code.)

Reply:
@ferdnyc - Yup, I considered deleting that multiple times already. Might as well just get rid of it.

Reply:
@ferdnyc - That debug file has been removed btw.